utils/src/main/java/com/redhat/lightblue/migrator/utils/DAOFacadeBase.java

```diff
@@ -31,12 +33,40 @@
 
     protected final D legacyDAO, lightblueDAO;
 
+    private EntityIdStore entityIdStore = null;
+
+    public EntityIdStore getEntityIdStore() {
+        return entityIdStore;
+    }
+
+    public void setEntityIdStore(EntityIdStore entityIdStore) {
+        this.entityIdStore = entityIdStore;
+
+        try {
+            Method method = lightblueDAO.getClass().getMethod("setEntityIdStore", EntityIdStore.class);
+            method.invoke(lightblueDAO, entityIdStore);
```

Comment:
But it does require a change, right? If they don't have this method implemented than... boom. Is there a scenario that would not have setEntityIdStore invoked?